/etc/gdm3/Xsession: Beginning session setup...
localuser:hacker being added to access control list
Failed to connect to the VirtualBox kernel service
Failed to connect to the VirtualBox kernel service
Failed to connect to the VirtualBox kernel service
Failed to connect to the VirtualBox kernel service
gnome-keyring-daemon: insufficient process capabilities, unsecure memory might get used
gnome-keyring-daemon: insufficient process capabilities, unsecure memory might get used
gnome-keyring-daemon: insufficient process capabilities, unsecure memory might get used
GNOME_KEYRING_CONTROL=/home/hacker/.cache/keyring-Yuc7ph
GPG_AGENT_INFO=/home/hacker/.cache/keyring-Yuc7ph/gpg:0:1
GNOME_KEYRING_CONTROL=/home/hacker/.cache/keyring-Yuc7ph
GPG_AGENT_INFO=/home/hacker/.cache/keyring-Yuc7ph/gpg:0:1
gnome-keyring-daemon: insufficient process capabilities, unsecure memory might get used
GNOME_KEYRING_CONTROL=/home/hacker/.cache/keyring-Yuc7ph
GPG_AGENT_INFO=/home/hacker/.cache/keyring-Yuc7ph/gpg:0:1
GNOME_KEYRING_CONTROL=/home/hacker/.cache/keyring-Yuc7ph
GPG_AGENT_INFO=/home/hacker/.cache/keyring-Yuc7ph/gpg:0:1
SSH_AUTH_SOCK=/home/hacker/.cache/keyring-Yuc7ph/ssh

(gnome-settings-daemon:4253): power-plugin-WARNING **: Failed set DPMS mode: Display is not DPMS capable

(gnome-settings-daemon:4253): color-plugin-WARNING **: failed to get edid: unable to get EDID for output

(gnome-settings-daemon:4253): PackageKit-WARNING **: failed to set proxy: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 246, in _method_lookup
    raise UnknownMethodException('%s is not a valid method of interface %s' % (method_name, dbus_interface))
UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: SetProxy is not a valid method of interface org.freedesktop.PackageKit


(gnome-settings-daemon:4253): PackageKit-WARNING **: failed to set root: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 246, in _method_lookup
    raise UnknownMethodException('%s is not a valid method of interface %s' % (method_name, dbus_interface))
UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: SetRoot is not a valid method of interface org.freedesktop.PackageKit


(gnome-settings-daemon:4253): updates-plugin-WARNING **: failed to set proxies: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 246, in _method_lookup
    raise UnknownMethodException('%s is not a valid method of interface %s' % (method_name, dbus_interface))
UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: SetProxy is not a valid method of interface org.freedesktop.PackageKit


(gnome-settings-daemon:4253): updates-plugin-WARNING **: failed to set install root: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 246, in _method_lookup
    raise UnknownMethodException('%s is not a valid method of interface %s' % (method_name, dbus_interface))
UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: SetRoot is not a valid method of interface org.freedesktop.PackageKit


(gnome-settings-daemon:4253): color-plugin-WARNING **: unable to get EDID for xrandr-default: unable to get EDID for output

(gnome-settings-daemon:4253): color-plugin-WARNING **: failed to reset xrandr-default gamma tables: gamma size is zero

(gnome-panel:4283): Gtk-CRITICAL **: gtk_accelerator_parse_with_keycode: assertion `accelerator != NULL' failed

** (gnome-panel:4283): WARNING **: Unable to parse mouse modifier '(null)'

Initializing tracker-miner-fs...
Tracker-Message: Setting up monitor for changes to config file:'/home/hacker/.config/tracker/tracker-miner-fs.cfg'
Starting log:
  File:'/home/hacker/.local/share/tracker/tracker-miner-fs.log'
Initializing tracker-store...
vmware-user: could not open /proc/fs/vmblock/dev
Tracker-Message: Setting up monitor for changes to config file:'/home/hacker/.config/tracker/tracker-store.cfg'
Tracker-Message: Setting up monitor for changes to config file:'/home/hacker/.config/tracker/tracker-store.cfg'
Starting log:
  File:'/home/hacker/.local/share/tracker/tracker-store.log'

(gnome-panel:4283): Gtk-CRITICAL **: gtk_accelerator_parse_with_keycode: assertion `accelerator != NULL' failed

** (gnome-panel:4283): WARNING **: Unable to parse mouse modifier '(null)'

Unable to open desktop file /usr/share/applications/zaproxy.desktop for panel launcher: No such file or directory
CachingBackend: Loading instances from cache
CachingBackend: Loading <Notes1>
[1398363648,000,xklavier.c:xkl_engine_start_listen/] 	The backend does not require manual layout management - but it is provided by the application
[1398363649,000,gkbd-status.c:gkbd_status_set_current_page_for_group/] 	Page for group 0 is not ready
[1398363649,000,gkbd-status.c:gkbd_status_set_current_page_for_group/] 	Page for group 0 is not ready
[1398363649,000,gkbd-status.c:gkbd_status_set_current_page_for_group/] 	Page for group 0 is not ready
=== Updating for primary Configuration ===
  Clone: false
  Output: Unknown attached to default
     status: on
     width: 1440
     height: 900
     rate: 60
     primary: true
     position: 0 0
Initializing nautilus-gdu extension
** Message: applet now removed from the notification area

(gnome-panel:4283): GLib-GObject-WARNING **: /build/buildd-glib2.0_2.33.12+really2.32.4-5-i386-eISom6/glib2.0-2.33.12+really2.32.4/./gobject/gsignal.c:2459: signal `size_request' is invalid for instance `0x8499ba0'
** Message: applet now embedded in the notification area
daemon already started

(gnome-settings-daemon:4253): color-plugin-WARNING **: unable to get EDID for xrandr-default: unable to get EDID for output

(gnome-settings-daemon:4253): color-plugin-WARNING **: failed to reset xrandr-default gamma tables: gamma size is zero

(gnome-settings-daemon:4253): color-plugin-WARNING **: unable to get EDID for xrandr-default: unable to get EDID for output

(gnome-settings-daemon:4253): color-plugin-WARNING **: failed to reset xrandr-default gamma tables: gamma size is zero

(gnome-settings-daemon:4253): color-plugin-WARNING **: unable to get EDID for xrandr-default: unable to get EDID for output

(gnome-settings-daemon:4253): color-plugin-WARNING **: failed to reset xrandr-default gamma tables: gamma size is zero
Found a running session of Notes, adding new instance by service.
Error in screenlets.services.get_service_by_name: org.freedesktop.DBus.Error.ServiceUnknown: The name org.screenlets.Notes was not provided by any .service files
Screenlet has already been added to /tmp/screenlets/screenlets.hacker.running
Loading instances in: /home/hacker/.config/Screenlets/Notes/default/
File: Notes1.ini
/home/hacker/.config/Screenlets/Notes/default/Notes1
LOAD NEW THEME: default
FOUND: /usr/share/screenlets/Notes/themes/default
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
Set options in NotesScreenlet
LOAD NEW THEME: tango
FOUND: /usr/share/screenlets/Notes/themes/tango
Screenlet has been initialized.
Restored instances from session 'default' ...
Window manager warning: CurrentTime used to choose focus window; focus window may not be correct.
Window manager warning: Got a request to focus the no_focus_window with a timestamp of 0.  This shouldn't happen!
** Message: applet now removed from the notification area

(update-notifier:4315): GConf-WARNING **: Got Disconnected from DBus.


(nm-applet:4312): GConf-WARNING **: Got Disconnected from DBus.


(tracker-miner-fs:4310): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.11 of volume monitor org.gtk.Private.GduVolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(tracker-miner-fs:4310): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.12 of volume monitor org.gtk.Private.GPhoto2VolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(tracker-miner-fs:4310): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.13 of volume monitor org.gtk.Private.AfcVolumeMonitor disconnected from the bus; removing drives/volumes/mounts
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
update-notifier: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.
gopenvpn: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

(gdu-notification-daemon:4333): Gdk-WARNING **: gdu-notification-daemon: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.

Received signal:15->'Terminated'
OK


(nautilus:4324): Gdk-WARNING **: nautilus: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

